guides/introduction/quick_start_guide.md

```diff
@@ -72,24 +75,24 @@ You can deploy the latest stable build of MeiliSearch straight on Heroku.
   </a>
 </p>
 
-The deploy can take up to 20 minutes because it will compile the whole project from the GitHub repository.
+Since all the source code from the GitHub repository will be compiled, the deployment process can take up to 20 minutes to complete.
```

Comment:
I think the whole heroku part can be removed.

Reply:
Removed!